cisco_secure_web_appliance/README.md

```diff
@@ -0,0 +1,274 @@
+
+    ```yaml
+      logs:
+      - type: file
+        path: <Path to Directory Where L4TM Logs would Get Stored>
+        service: l4tm_logs
+        source: cisco-secure-web-appliance
+      logs:
+      - type: udp
+        port: <PORT>
+        service: access_logs
+        source: cisco-secure-web-appliance
+    ```
+    **Note**: It is important not to change the service and source values, as these parameters are integral to the pipeline's operation.
+    
+3. [Restart the Agent][3].
+
+
+#### Steps to configure Syslog Push for Access Logs:
+
+**Prerequisites:**
+1. Require Syslog server hostname(Your Datadog Agent Server) where you want to push the logs.
+
+**Configuration:**
+
+1. Login to Cisco Secure Web Appliance UI.
+2. Navigate to  System Administration -> Log Subscriptions.
+3. In order to add  Access Logs subscription, click on the Add `Log Subscription` button.
+4. Select Log Type as Access Logs.
+5. Provide Log Name.
+6. Choose `Squid` Option for Log Style.\
+    **Note**: We will support default (squid) log style for access log. 
+7. Select `Syslog Push` Option as Retrieval Method.
+8. Provide Following Details.
+
+    Hostname: \<Datadog-Agent Host Server>
+
+    Port: \<Default Provided>
+
+    Protocol: UDP
+    
+    Maximum message size: \<Valid values for UDP are 1024 to 9216>
+    
+    Facility: \<Default Selected>
+9. Click on Submit. 
+10. Click on Commit Changes at top right-side of Log Subscriptions Page.
+    NOTE: These changes will not go into effect until you commit them.
+
+#### Steps to configure SCP on Remote Server for L4TM Logs.
+
+**Prerequisites:**
+1. Requires SCP Host and Username for the same.
+
+**Configuration:**
+1. Navigate to  System Administration -> Log Subscriptions in Cisco Secure Web Appliance UI.
+2. To add a log subscription for Traffic Monitor Logs, click on  Add `Log Subscription`.  
+3. Select Traffic Monitor Logs as Log Type.
+4. Provide appropriate Log Name.
+5. For FileName, provide a new name or keep the default added name.
+6. Choose SCP on Remote Server as Retrieval Method.
+7. Provide Following Information.
+
+    SCP Host: \<SCP Host IP Address>
+
+    Directory: \<Path to Directory Where Logs would Get Stored>
+    
+    **NOTE:** Make sure that Directory does not have any other log files.
+    SCP Port: \<Default Port>
+    Username: \<SCP Host Username>
+8. Click on Submit. After submitting, SSH key(s)  will get generated. Copy and save both the SSH key(s) as it is only visible once.
+9. Place the given SSH key(s) into your `authorized_keys` file on the remote host so that the log files can be uploaded.
+10. Click on Commit Changes at top right-side of Log Subscriptions Page.
+
+    **NOTE:** These changes will not go into effect until you commit them.
+
+#### Steps to configure SCP on Remote Server for Access Logs.
+
+**Prerequisites:**
+1. Requires SCP Host and Username for the same.
```

Comment:
Not sure what this means.
Does it mean the you need the SCP Host and its admin account username?

Reply:
Understood your confusion. this basically means that it requires hostname and username (admin account username is not necessary) of VM/machine on which datadog-agent is installed. Also we have updated the sentence accordingly.